starboard/shared/win32/socket_internal.h

```diff
@@ -69,7 +69,7 @@ struct SbSocketPrivate {
   // edge-triggered, unlike other events.
   //
   // See MSDN documentation for WSAEventSelect FD_WRITE for more info.
-  starboard::atomic_bool writable;
+  std::atomic_bool writable;
```

Comment:
Please default-initialize. Doublecheck the implementation for correct value, i'm unsure

Reply:
Done